.github/workflows/build_msi_installer.yml

```diff
@@ -98,6 +91,24 @@ jobs:
           MSI_PRIVATE_VERSION: ${{ github.event.inputs.set_msi_private_version }}
         shell: pwsh
 
+      - name: Update promptflow package version when set msi private version
```

Comment:
is there any trigger? or succeeded pipeline? please attach one in your description.

Reply:
yeah, have validated.